### DISCUSSION TITLE
Protocol not supported [93]

### DISCUSSION THREAD

**Discussion by DalvinderSinghSond** [baresip/baresip#2570] [Q&A] [answered]
Hi
I have moved from
libre v2.11.0
librem v2.11.0
baresip v2.11.0

to
libre 3.1.0
baresip 3.1.0

after that I have problem with SIP communication.
When I use 
/uanew <sip:01111@10.70.149.125>
Creating UA for <sip:01111@10.70.149.125> ...
I get
ua: SIP register failed: Protocol not supported [93]

I use
OpenSSL 1.1.1n

is there ny mismatch between libraries after moving to latest lire and baresip?

I someone can help me.
Regards
Singh

**Comment by DalvinderSinghSond** [comment]
I get some error start baresip
build/baresip
baresip v3.1.0 Copyright (C) 2010 - 2022 Alfred E. Heggestad et al.
Local network addresses:
 enp57s0u1u4:  10.70.149.125
       wlo1:  192.168.15.156
     virbr0:  192.168.122.1
 enp57s0u1u4:  fe80::2fe8:93a4:1729:e000
       wlo1:  2001:840:4b4f:beef:f8b7:4aa4:767a:23b0
       wlo1:  fe80::de13:a1e2:a7a0:6ca4
ua: adding SIP CA file: /etc/ssl/certs/ca-certificates.crt
ua: adding SIP CA path: 
ua: tls_add_ca() failed: No such file or directory [2]
ua: wss tls_add_ca() failed: No such file or directory [2]
aucodec: PCMU/8000/1
aucodec: PCMA/8000/1
aufilt: auconv
aufilt: auresamp
ausrc: alsa
auplay: alsa
medianat: stun
medianat: turn
medianat: ice
Populated 0 accounts
account: No SIP accounts found
 -- check your config or add an account using 'uanew' command
Populated 7 contacts
Populated 2 audio codecs
Populated 2 audio filters
Populated 0 video codecs
Populated 0 video filters
baresip is ready.

**Comment by DalvinderSinghSond** [comment]
I have solved the issue....my laptop got new IP address from DHCP...and that not match 
sip_listen		10.70.149.125:5060

in config file

**Reply by cspiel1** [reply]
You don't have to specify the IP address. To set the local listen port it is enough to set:
```
sip_listen		0.0.0.0:5060
```

**Reply by DalvinderSinghSond** [reply, marked as answer]
Thank for your advice!
Better to use
sip_listen		0.0.0.0:5060
